doc/check.texi

```diff
@@ -1032,6 +1032,51 @@ respectively.
 Similar to @code{ck_assert_int_*}, but compares two unsigned integer values
 (@code{uintmax_t}) instead.
 
+@item ck_assert_floating_eq
+@itemx ck_assert_floating_ne
+@itemx ck_assert_floating_lt
+@itemx ck_assert_floating_le
+@itemx ck_assert_floating_gt
+@itemx ck_assert_floating_ge
+
+Compares two floating point numbers (@code{float}, @code{double},
+@code{long double}) and displays predefined message with condition and
```

Comment:
...and displays a predefined message with both the condition and input parameters on failure.